styles.css

```diff
@@ -0,0 +1,60 @@
+
+/* ......... the header section.......... */
+header {
+  width: 375px;
+  height: 48px;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-around;
+  color: #fff;
+  padding-top: 44px;
+}
+
+.name {
+  font-family: 'Roboto', sans-serif;
+  line-height: 24px;
+}
+
+/* ............headline section............. */
+
+.headline {
```

Comment:
- [ ] Good work adding a display flex to the `ul` of social icons. Kindly make them smaller to look similar to the FIGMA template

![image](https://user-images.githubusercontent.com/93630700/205456469-642c4421-abbf-41ad-93cf-19b3ab5c1230.png)